conans/util/windows.py

```diff
@@ -69,7 +71,9 @@ def path_shortener(path, short_paths):
         # cmd can fail if trying to set ACL in non NTFS drives, ignoring it.
         pass
 
-    redirect = tempfile.mkdtemp(dir=short_home, prefix="")
+    redirect = hashed_redirect(short_home, path) or tempfile.mkdtemp(dir=short_home, prefix="")
+    save(os.path.join(redirect, CONAN_REAL_PATH), path)
```

Comment:
I can't see when this file is used. Shouldn't this file be used by `hased_redirect()` to see if the shortened path correspond already to a reference or something similar?
Imagine that I remove by hand the `~/.conan/data` but the shortened folders are kept. Next thing I try to create the short path, it could collide and create `tmp_folder`. If we want that, why is this file necessary? Otherwise, is it file used?

Reply:
Ok, then please add a comment saying that the file is to debugging purposes only and not used by conan directly.

Reply:
I added a comment.